katacomb/katacomb/conf/MKAT_narrow_L.yaml

```diff
@@ -0,0 +1,60 @@
+# Default L-band MFImage configuration options for narrowband 'online' mode
+#
+# These parameters are the defaults used by the `continuum_pipeline.py` script
+# for online processing in the SDP pipeline. The online parameters are
+# optimised to produce images within twice the observing time and CLEAN
+# components suitable for continuum subtraction by the spectral imager.
+#
+# Further details of the parameter selection can be found in JIRA ticket
+# SPR1-1332: https://skaafrica.atlassian.net/browse/SPR1-1332
```

Comment:
This jira ticket doesn't seem to include any of Isaac's more recent work on unifying the 50 and 100 MHz cases. If a ticket does exist for this it would be useful to also mention it here.